package.json

```diff
@@ -75,13 +75,14 @@
     "@babel/runtime": "^7.9.2",
     "@carbon/charts": "^0.30.5",
     "@carbon/charts-react": "^0.30.5",
-    "@carbon/colors": "10.7.0",
-    "@carbon/icons": "10.6.1",
-    "@carbon/icons-react": "10.6.1",
+    "@carbon/colors": "10.8.0",
+    "@carbon/icons": "10.9.3",
+    "@carbon/icons-react": "^10.10.0",
```

Comment:
do we need to use ^ for the icons-react?  shouldn't we lock it?

Reply:
yes. thanks